ENIGMAsystem/SHELL/Graphics_Systems/Direct3D11/DX11textures.cpp

```diff
@@ -40,16 +39,11 @@ int graphics_create_texture(const RawImage& img, bool mipmap, unsigned* fullwidt
   
   *fullwidth  = nlpo2dc(img.w)+1;
   *fullheight = nlpo2dc(img.h)+1;
-  
-  if (img.pxdata != nullptr && (img.w != *fullwidth || img.h != *fullheight)) {
-    RawImage padded = image_pad(img, *fullwidth, *fullheight);
-    tbsd.pSysMem = padded.pxdata;
-  } else tbsd.pSysMem = img.pxdata;
-  
-  tbsd.SysMemPitch = *fullwidth*4;
+
+  tbsd.SysMemPitch = (*fullwidth)*4;
   // not needed since this is a 2d texture,
   // but we can pass size info for debugging
-  tbsd.SysMemSlicePitch = *fullwidth*(*fullheight)*4;
+  tbsd.SysMemSlicePitch = (*fullwidth)*(*fullheight)*4;
```

Comment:
I'd have added spacing around the multiplication rather than / in addition to parenthesizing the dereference.

Reply:
Done f8cc73f6ae376c23fb6de0218aa30de76547b8ce